src/networks.js

```diff
@@ -3,7 +3,6 @@
 
 module.exports = {
   bitcoin: {
-    magic: 0xd9b4bef9,
```

Comment:
We haven't stored all the `chainParams` that exist [here](https://github.com/bitcoin/bitcoin/blob/master/src/chainparams.cpp#L86-L89).
It seems inconsistent to just have this one,  which feels very out of place among constants that are actually used and tested by this library.

Open to ideas on this one,  removing it for now.  Maybe it should come back as `chain: { magicPrefix: ? }}`?